### PR TITLE
BGE-4 - Adding a README.md 

### PR DESCRIPTION
**Pull request by Marion3531**
Adding the README.md file to the project.

### CODE REVIEW

**Review by Lixiu520 — APPROVED** [review]
perfect!

**Review by camilleroinet — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+HTML, CSS, JS, PHP
+
+***Files***
+about-us.html, contact.html, contact.php, features.html, index.html, portofolio.html, pricing.html, services.html, style.css, testimonials.html
+
+***Directory***
+.git, css, fonts, images, js, uploads
+
+***Credits***
+Marion & Constant lpb
```
Comment (lena-my) [inline README.md:18]: lpb = les plus beaux

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+SIGSCAN
+
+***Project description***
+website of a tracking company
+Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium suscipit, esse maxime quidem corporis dolorem necessitatibus ducimus minima blanditiis ipsam hic fuga commodi molestias nesciunt porro voluptatibus tempore repellat magnam.
+
+***Language***
+HTML, CSS, JS, PHP
+
+***Files***
```
Comment (diziete) [inline README.md:11]: Good job

**Review by fabienmarotte — APPROVED** [review]

**Review by NajehRhaiem — APPROVED** [review]

**Review by diziete — APPROVED** [review]
Good job

**Pull request merged**